site/public/js/rainbow-customization.js

```diff
@@ -413,6 +413,38 @@ function getBenchmarkPercent() {
     return benchmark_percent;
 }
 
+function getFinalBenchmarkPercent() {
```

Comment:
![스크린샷 2024-06-17 오전 10 28 25](https://github.com/Submitty/Submitty/assets/123261952/8e742d79-d525-4537-bed9-d99ebdb6ddc2)

When I un-check final_cutoff and try to hit save changes (which means I don't want the final cutoff in my customization.json) it still would not save insisting "All final cutoffs must have a value before saving"

This needs to be addressed.

Reply:
should be fixed by [ab6358e](https://github.com/Submitty/Submitty/pull/10518/commits/ab6358e12bed099d941fe1a6bdd39d5df53bc606)

Reply:
Default cutoff values will be included in the JSON when the checkbox is unselected, but they should be unused with no effect on RainbowGrades.